script/makefiles/site.mk

```diff
@@ -0,0 +1,13 @@
+# Copyright 2021-2022 the Kubeapps contributors.
```

Comment:
```suggestion
# Copyright 2022 the Kubeapps contributors.
```

Reply:
This is a copy&paste from another script in the same file, I think we should be consistent and, in case we agreed on changing this, we should do it for every file in the repo.

@antgamdia Do you agree to merge it as is and handle this task in another issue?

Reply:
No, no, it is just a one-time change for this specific file. The policy we are following (after checking with the legal counsel office) is using the date on which the file was created and the date of the more recent (significant) change.
In case of fresh new files, like this one, both dates match, so, to avoid redundancies, we shorten `2022-2022` with just `2022`, wherefore my suggestion.

Reply:
Ok, I understand. Let's apply the suggestion then. Thanks for the explanation!